nservicebus/upgrades/host-6to7.md

```diff
@@ -47,4 +47,17 @@ WARNING: If an `EndpointConfig.cs` file already exists in the project, be carefu
 
 The WCF integration using `WcfService` has been moved from the host to the separate NuGet package [NServiceBus.Wcf](https://www.nuget.org/packages/NServiceBus.Wcf/). That package must be used in order to use the WCF integration functionality when targeting NServiceBus versions 6 and above. The NServiceBus.Wcf NuGet package **has no dependency** on the NServiceBus.Host NuGet package and can also be used in self-hosting scenarios.
 
-The WCF integration has been augmented with additional functionality such as the ability to reply with messages to the client, cancel requests and reroute to other endpoints. More information can be found in [Wcf](/nservicebus/wcf).
\ No newline at end of file
+The WCF integration has been augmented with additional functionality such as the ability to reply with messages to the client, cancel requests and reroute to other endpoints. More information can be found in [Wcf](/nservicebus/wcf).
+
+### Ambiguous type compilation error
+
+By default referenced assemblies are imported into the global namespace. This might lead to ambiguous type reference problems when implementing `WcfService<TRequest, TResponse>` or `IWcfService<TRequest, TResponse>`. To resolve the ambiguous type reference do the following:
```

Comment:
Should we add a link for global namespace, as in [global namespace](https://msdn.microsoft.com/en-us/library/z2kcy19k.aspx#Anchor_0)? People might think it's something else.